app/models/component/manifest/app_container.rb

```diff
@@ -37,6 +37,7 @@ def environment_settings
         env << { name: '_JAVA_OPTIONS', value: java_options.to_s }.with_indifferent_access
         env << { name: '_BUSBAR_BUILD_TIME', value: timestamp.to_s }.with_indifferent_access
         env << { name: 'PORT', value: app_port.to_s }.with_indifferent_access
+        env << { name: 'DD_AGENT_HOST', valueFrom: fieldRef: fieldPath: 'status.hostIP'}.with_indifferent_access
```

Comment:
is this valid Ruby? 
I haven't seen `valueFrom: fieldRef: fieldPath: 'status.hostIP'` in Ruby before

Reply:
Actually... I made an update and forgot to push... just a sec! XD